### PR TITLE
Endpoint for Jest

### PR DESCRIPTION
**Pull request by alexkilgour**
Forgot to add endpoint for Jest

whatru

### CODE REVIEW

File: jest.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('./configurations/jest');
```
Comment (sonniesedge) [inline jest.js]: No trailing space. Is this linted? 😂

**Comment by sonniesedge** [comment]
Top quality giffing.

**Comment by sonniesedge** [comment]
"Add CoCo to the projects" alex u r drunk

**Comment by alexkilgour** [comment]
Yeah, i know, i know. So many mistakes today

**Pull request merged**